scripts/circle-ci/publish-to-antithesis.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+printenv ANTITHESIS_LOGIN_JSON | base64 -d | docker login -u _json_key https://${ANTITHESIS_REPO_URL} --password-stdin
```

Comment:
We should probably check the variables are defined? It's probably fine but might be nicer to fail gracefully